components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/API.java

```diff
@@ -164,7 +164,7 @@ public boolean isDefaultVersion() {
         return tags;
     }
 
-    public Set<String> getLabels() {
+    public List<String> getLabels() {
```

Comment:
We have used Sets to avoid duplicates. Any reason to use List here? Better to stick with sets.